### ISSUE TITLE
maxlength support

### ISSUE THREAD

**Issue by arthedza** [thegamenicorus/react-native-phone-input#151]
For example:
```
<PhoneInput
    maxLength={14}
    ...
```

**Comment by SirPhemmiey** [comment]
Hey @arthedza did you find a fix for this?

**Comment by arthedza** [comment]
> Hey @arthedza did you find a fix for this?

No, but I've found a [PR](https://github.com/thegamenicorus/react-native-phone-input/pull/74) for maxLength support.

**Comment by andrushkodmytro** [comment]
Hi. You can pass maxLength in textProps. 
<ReactNativePhoneInput
        ref={inputRef}
        initialValue={value}
        style={input}
        offset={30}
        textStyle={textStyle}
        initialCountry={'no'}
        textProps={maxLength: 10}
         onPressFlag={() => {
          handleOpenDropdown();
        }}
        onChangePhoneNumber={onChangePhoneNumber}
      ></ReactNativePhoneInput>

**Comment by bharvadiyavishal** [comment]
> 

not working

**Comment by bharvadiyavishal** [comment]
how can i maxlength(10) in react-native-phone-number-input.

**Comment by marquinmpfs** [comment]
You can use a prop:

`textInputProps={{ maxLength: 12 }}`

**Comment by codegien** [comment]
This works

**Comment by codegien** [comment]
This works:
textInputProps={{maxLength: 12}}
